apps/jobs/serializers.py

```
@@ -60,11 +55,53 @@ class Meta:
+    def validate_corp_id(self, value):
+        # 해당 기업이 실제로 존재하는지 검증합니다.
+        if not Corp.objects.filter(id=value, is_deleted=False).exists():
+            raise serializers.ValidationError("해당 기업을 찾을 수 없습니다.")
+        return value
+
+    def validate(self, data):
+        # 이미 즐겨찾기에 추가했는지 중복 여부를 검증합니다.
+        user = self.context['request'].user
+        corp_id = data.get('corp_id')
+        if CorpBookmark.objects.filter(user=user, corp_id=corp_id).exists():
+            raise serializers.ValidationError("이미 즐겨찾기에 추가된 기업입니다.")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)